soc/xtensa/nxp_adsp/Kconfig

```diff
@@ -14,6 +14,10 @@ config SOC_FAMILY
 	string
 	default "nxp_adsp"
 
+config NXP_ADSP_SIGN_IMAGE
```

Comment:
hi @iuliana-prodan , if you don't sign NXP images, can we simply remove the custom command from nxp_adsp/CMakeLists.txt and not add the NXP_ADSP_SIGN_IMAGE config?

Reply:
Hi @dluke62,

I know it seems reasonable to remove it if not used, but we added to be in sync with Intel, since the DSP (at least for nxp_adsp_imx8m and nxp_adsp_imx8) is used for SOF (https://thesofproject.github.io/latest/architectures/firmware/sof-zephyr/index.html). We are not signing the images now, but we might.

Please, let's keep it, for now. 
If we remove it, we need more testing for all platforms.

Also, just FYI, for nxp_adsp_imx8m and  nxp_adsp_imx8 we are using rimage to create the SOF firmware (from the zephyr elf we add some custom headers and we obtain a .ri image).